### ISSUE TITLE
<kbd> has a black background

### ISSUE THREAD

**Issue by garrett** [cockpit-project/cockpit#8904]
`<kbd>` has semantic meaning and should be used when someone is expected to type a command. However, it's styled up inappropriately within Cockpit (possibly from PatternFly).

First spotted in https://github.com/cockpit-project/cockpit/pull/8785#discussion_r177353725; you can see the issue in this screenshot:

![screenshot of kbd in action](https://user-images.githubusercontent.com/3228183/37957336-598f2408-31b6-11e8-92fc-e6e6e32e925e.png)

**Comment by garrett** [comment]
This style gets pulled in Bootstrap (as PatternFly builds on Bootstrap):

https://github.com/patternfly/patternfly-site/blob/3d38a313065aa2e8ea91b68a3377a0fcb94869b8/library/components/patternfly/less/lib/bootstrap/code.less#L24

I wasn't able to find it in the PatternFly source repos, because it's not there. But it is in the PatternFly site repo (linked above).

**Comment by garrett** [comment]
Opened an issue @ https://github.com/patternfly/patternfly/issues/1002

**Comment by garrett** [comment]
I think we can safely use the CSS unset value here, as all browsers that support grid also support unset.

Compare:
1. https://caniuse.com/#search=grid
2. https://caniuse.com/#feat=css-unset-value

The first number in green is when the feature was introduced. The number is higher in grid and supported everywhere, so it's safe for us to use unset.

This is great, as we can undo PatternFly styles without having to do one-offs or require PatternFly to fix it upstream.

**Comment by garrett** [comment]
HOWEVER, in this instance, we dropped `<kbd>` usage in the PR.

And it seems `<kbd>` has no support in screen readers or in the Firefox accessibility tree, according to https://stackoverflow.com/questions/15190247/does-the-kbd-element-help-at-all-in-terms-of-accessibility.

As a result, I'm simply closing this issue. If it comes up again, and we determine we should use it, we can use unset.
